homeassistant/components/recorder/util.py

```diff
@@ -34,6 +40,12 @@
 # should do a check on the sqlite3 database.
 MAX_RESTART_TIME = timedelta(minutes=10)
 
+# Retry when one of the following MySQL errors occurred:
+RETRYABLE_MYSQL_ERRORS = (1205, 1206, 1213)
```

Comment:
We should be consistent in our spelling.

Reply:
Fixed in c29740f